### PR TITLE
MC PID Tuning - Fixes  

### PR DESCRIPTION
**Pull request by hamishwillee**
Follow up to #1094

Covers
- RPM controllers don't need thrust tuning
- Simple position control in UI

@bkueng Is this daily build only? My stable build 4.1. something doesn't have it.

### CODE REVIEW

File: en/config_mc/pid_tuning_guide_multicopter_basic.md [hunk 1]
```diff
@@ -67,10 +67,16 @@ The tuning procedure is:
...
-   The thrust curve value can be used to compensate for this non-linearity (for more information see the [detailed PID tuning guide](../config_mc/pid_tuning_guide_multicopter.md#thrust-curve)
+   For PWM, power-based and (some) UAVCAN speed controllers, the control signal to thrust relationship may not be linear.
+   As a result, the optimal tuning at hover thrust may not be ideal when the vehicle is operating at higher thrust.
+   
+   The thrust curve value can be used to compensate for this non-linearity (for more information see the [detailed PID tuning guide](../config_mc/pid_tuning_guide_multicopter.md#thrust-curve)).
    A value of 0.3 is a good default.
    :::
+   
+   :::tip
+   RPM-based controllers and some UAVCAN ESC have a linear thrust relationship so you do not need to tune the thrust curve.
```
Comment (hamishwillee) [inline en/config_mc/pid_tuning_guide_multicopter_basic.md:78]: @bkueng Can you please check this whole bullet point. Want to make sure I have captured that it is to do with the control-signal to thrust linearity.

Question - what happens if you use an RPM based controller? I assume you still have to tune it, just not for thrust. Do you just then use 0 for the thrust curve?

File: en/config_mc/pid_tuning_guide_multicopter_basic.md [hunk 2]
```diff
@@ -107,7 +114,10 @@ The tuning procedure is:
      :::
 1. Repeat the tuning process for the attitude controller on all the axes.
 1. Repeat the tuning process for the velocity and positions controllers (on all the axes).
-   Use Position mode when tuning these controllers (make sure to switch to the simple input mode, allowing direct control so that you can generate step inputs).
+   - Use Position mode when tuning these controllers
+   - Select the **Simple position control** option in the *Position control mode ...* selector (this allows direct control for the generation of step inputs)
```
Comment (hamishwillee) [inline en/config_mc/pid_tuning_guide_multicopter_basic.md:118]: FYI this highlights the simple position control option in the UI

File: en/config_mc/pid_tuning_guide_multicopter_basic.md (hunk 1 above)
Comment (bkueng) [reply en/config_mc/pid_tuning_guide_multicopter_basic.md:79]: For RPM, it's squared, so you would set it to 1 (no tuning needed)

**Review by bkueng — COMMENTED** [review]
>   @bkueng Is this daily build only? My stable build 4.1. something doesn't have it.

Correct, it's not in a release yet

**Comment by hamishwillee** [comment]
Thanks @bkueng . I added info re value for RPM and also which QGC to use. Merged.

**Pull request merged**